setup.cfg

```diff
@@ -28,6 +28,11 @@ max-line-length = 80
 max-complexity = 18
```

Comment:
Actually...now that I come to think about this...Why do we need the setup.cfg in the template repo? Shouldn't this file be only generate in the initialization?